tests/meta/demultiplex/test_validation.py

```diff
@@ -19,45 +19,45 @@ def test_validate_sample_fastq_with_valid_file():
     sample_fastq = Path("Sample_123/sample_L0002.fastq.gz")
 
     # WHEN validating the sample fastq file
-    validate_sample_fastq_file(sample_fastq)
+    is_valid = is_valid_sample_fastq_file(sample_fastq=sample_fastq, sample_internal_id="sample")
 
-    # THEN no exception should be raised
+    # THEN it should be valid
+    assert is_valid
 
 
 def test_validate_sample_fastq_without_sample_id_in_parent_directory_name():
-    # GIVEN a sample fastq file without a sample id in the parent directory name
-    sample_fastq = Path("sample_L0002.fastq.gz")
+    # GIVEN a sample fastq file without a sample id in the parent directory name or file name
+    sample_fastq = Path("L0002.fastq.gz")
 
     # WHEN validating the sample fastq file
-    # THEN a ValueError should be raised
-    with pytest.raises(
-        ValueError, match="Parent directory name of sample fastq must contain 'Sample_<sample_id>'."
-    ):
-        validate_sample_fastq_file(sample_fastq)
+    is_valid_fastq = is_valid_sample_fastq_file(
+        sample_fastq=sample_fastq, sample_internal_id="sample_id"
+    )
+
+    # THEN it should not be valid
+    assert not is_valid_fastq
 
 
 def test_validate_sample_fastq_without_lane_number():
     # GIVEN a sample fastq file without a lane number
-    sample_fastq = Path("Sample_123/sample.fastq.gz")
+    sample_fastq = Path("Sample_123/sample_id.fastq.gz")
 
     # WHEN validating the sample fastq file
-    # THEN a ValueError should be raised
-    with pytest.raises(
-        ValueError, match="Sample fastq must contain lane number formatted as '_L<lane_number>'."
-    ):
-        validate_sample_fastq_file(sample_fastq)
+    is_valid_fastq = is_valid_sample_fastq_file(sample_fastq, sample_internal_id="sample_id")
+
+    # THEN it should not be valid
+    assert not is_valid_fastq
 
 
 def test_validate_sample_fastq_without_fastq_file_extension():
     # GIVEN a sample fastq file without a .fastq.gz file extension
-    sample_fastq = Path("Sample_123/sample_L0002.fastq")
+    sample_fastq = Path("Sample_123/123_L0002.fastq")
```

Comment:
This is clearly confusing from the tittle as it says fastq file extension not fastq.gz

Reply:
Yeah, I feel that "test_validate_non-compressed_sample_fastq..." might be a better name.

Reply:
Good point, changing it